<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMeansDialog</class>
 <widget class="QDialog" name="KMeansDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>193</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>k-Means</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>k-Means</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="partitionLabel">
     <property name="text">
      <string>Partition Policy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="partitionComboBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Sample Initialization</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random Partition</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Refined Start</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="metricLabel">
     <property name="text">
      <string>Distance Metric</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="metricComboBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Squared Euclidean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Euclidean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chebyshev</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manhattan</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="clustersSpinBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="clusterLabel">
     <property name="text">
      <string>Clusters</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="emptyCheckBox">
     <property name="text">
      <string>Allow Empty Clusters?</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KMeansDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KMeansDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
